src/main/java/org/opentripplanner/middleware/tripmonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -428,10 +428,13 @@ private void sendNotifications() {
         Map<String, Object> templateData = Map.of(
             "tripId", trip.id,
             "tripName", trip.tripName,
-            "notifications", notifications.stream()
+            "notifications", new ArrayList<>(notifications).stream()
                 // Make certain notifications, such as the initial reminder one, appear on top.
                 .sorted(Comparator.comparingInt(TripMonitorNotification::sortOrder))
-                .map(notification -> notification.body)
+                .map(notification -> Map.of(
+                        "type", notification.type.toString(),
+                        "body", notification.body
+                ))
                 .collect(Collectors.toList())
```

Comment:
Haha I think this should be simply:
```suggestion
            "notifications", new ArrayList<>(notifications)
```